### PR TITLE
Add docs - schemas, connectors, acls, index

### PR DESCRIPTION
**Pull request by muralibasani**
Docs added for
- Schemas
- Connectors
- Acls
- Support section in home page

### CODE REVIEW

File: docs/docs/howto/connectors/My-connector-requests.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+My Connector Requests
+=====================
+
+The **My Connector Requests** page can be accessed through the **Requests -> My Connector Requests** navigation bar. You can view all requests created by you here. This page also provides an audit log and a history of the connector requests created by you.
+
```
Comment (aindriu-aiven) [inline docs/docs/howto/connectors/My-connector-requests.rst:5]: Is this My or is it My/My Team's requests
I think you can view all the requests by your team right?

File: docs/docs/howto/kafka-cluster-migration/sync-connectors-from-cluster.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Synchronize Topics from cluster
+===============================
```
Comment (aindriu-aiven) [inline docs/docs/howto/kafka-cluster-migration/sync-connectors-from-cluster.rst:2]: Just need to update this header

File: docs/docs/howto/subscriptions/My-subscription-requests.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The Requests are displayed in chronological order by 'Date Requested' with the most recent request at the top and the oldest request at the bottom. Additionally, on this page, you can find any request you and your team have made and its current status.
+
+Filtering the requests
+----------------------
+
+For easy navigation use the dropdown to filter the requests which you are viewing by the status of the request.
+
+- all
+   View all requests that you have created.
```
Comment (aindriu-aiven) [inline docs/docs/howto/subscriptions/My-subscription-requests.rst:14]: you and your team have created.

File: docs/docs/howto/connectors/My-connector-requests.rst (hunk 1 above)
Comment (muralibasani) [reply docs/docs/howto/connectors/My-connector-requests.rst:5]: yes, we can view all my team's requests

**Review by harshini-rangaswamy — APPROVED** [review]
Minor editorial changes done. The rest looks good.

**Pull request merged**